firefox-ios/Client/Frontend/Home/HomepageContextMenuProtocol.swift

```diff
@@ -55,6 +55,7 @@ extension HomepageContextMenuProtocol {
         with sourceView: UIView?,
         sectionType: HomepageSectionType
     ) -> PhotonActionSheet? {
+        guard let windowUUID = sourceView?.currentWindowUUID else { return nil }
```

Comment:
would it be safer to pass in windowUUID from the view controller if we know that's guaranteed?

Reply:
I'm not sure offhand but I will dig into this a bit further, going to add a ToDo to revisit this.